src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -1247,9 +1329,12 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
 
                 var headlines = result.HeadlinesSection;
                 Assert.Single(headlines.Content);
-
                 Assert.Equal(expectedContent, (headlines.Content[0] as HtmlBlockViewModel)?.Body);
 
+                var relatedDashboardsSection = result.RelatedDashboardsSection;
+                Assert.Single(relatedDashboardsSection.Content);
```

Comment:
I think you could combine a check to make sure that `relatedDashboardsSection` is not null here and supress the warning the appears to be present.

Same problem also on line 1100.